python/pyhail/context.py

```diff
@@ -81,6 +81,20 @@ def run_command(self, vds, pargs):
     def grep(self, regex, path, max_count=100):
         """Grep big files, like, really fast.
 
+        **Examples**
+
+        Print all lines containing the string ``hi`` in ``file.txt``:
+
+        >>> hc.grep('hi','file.txt')
+
+        Print all lines containing digits in ``file1.txt`` and ``file2.txt``:
+
+        >>> hc.grep('\d', ['file1.txt','file2.txt')
+
+        **Background**
+
+        ``grep`` mimics the basic functionality of Unix grep using the parallel environment of Spark to speed up queries by orders of magnitude.  Results are printed to the screen. This command is provided as a convenience to those in the statistical genetics community who often work with enormous text files like VCFs. Find background on regex at `RegExr <http://regexr.com/>`_.
+
         :param str regex: The regular expression to match.
 
         :param path: The files to search.
```

Comment:
It already says `:return: Nothing` since this command just prints to screen. What's the appropriate type?